src/app/@ansyn/ansyn/modules/core/services/credentials/credentials.service.ts

```diff
@@ -18,6 +18,7 @@ export class CredentialsService {
 
 	constructor(protected httpClient: HttpClient,
 				@Inject(credentialsConfig) public config: ICredentialsConfig) {
+		this.getCredentials().subscribe();
```

Comment:
this.getCredentials().pipe(take(1)).subscribe();

Reply:
getCredentials()  method should be defined as 
getCredentials(): Observable<boolean>

user(): string
authorizedAreas(): proper type
etc.